halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/config/v1/FileWriter.java

```diff
@@ -16,31 +16,21 @@
 
 package com.netflix.spinnaker.halyard.config.config.v1;
 
-import com.fasterxml.jackson.databind.ObjectMapper;
-import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
-import org.yaml.snakeyaml.Yaml;
 
 import java.io.*;
-import java.util.Map;
 
 @Component
-public class YamlWriter {
-  @Autowired
-  Yaml yamlParser;
-
-  @Autowired
-  ObjectMapper objectMapper;
-
-  public void write(Object node, String path) throws IOException {
-    write(node, path, false);
+public class FileWriter {
+  public void write(String contents, String path) throws IOException {
+    write(contents, path, false);
   }
 
-  public void write(Object node, String path, boolean append) throws IOException {
+  public void write(String contents, String path, boolean append) throws IOException {
```

Comment:
I am appending, and for a brief time it is possible that it's in an inconsistent state.